docs/data-sources/dms_rabbitmq_flavors.md

```diff
@@ -0,0 +1,143 @@
+---
+subcategory: "Distributed Message Service (DMS)"
+---
+
+# g42cloud_dms_rabbitmq_flavors
+
+Use this data source to get the list of RabbitMQ available flavor details within G42Cloud.
+
+## Example Usage
+
+### Query the list of RabbitMQ flavors by cluster type
+
+```hcl
+data "g42cloud_dms_rabbitmq_flavors" "test" {
+  type = "cluster"
+}
+```
+
+### Query the list of RabbitMQ flavors by flavor ID
+
+```hcl
+data "g42cloud_dms_rabbitmq_flavors" "test" {
+  flavor_id = "c6.2u4g.cluster"
+}
+```
+
+### Query the list of RabbitMQ flavors by availability zone
+
+```hcl
+variable "az1" {}
+variable "az2" {}
+
+data "g42cloud_dms_rabbitmq_flavors" "test" {
+  availability_zones = [
+    var.az1,
+    var.az2,
+  ]
+}
+```
+
+## Argument Reference
+
+* `region` - (Optional, String) Specifies the region in which to obtain the dms RabbitMQ flavors.
+  If omitted, the provider-level region will be used.
+
+* `flavor_id` - (Optional, String) Specifies the DMS flavor ID, e.g. **c6.2u4g.cluster**.
+
+* `storage_spec_code` - (Optional, String) Specifies the disk IO encoding.
+  + **dms.physical.storage.high.v2**: Type of the disk that uses high I/O.
+  + **dms.physical.storage.ultra.v2**: Type of the disk that uses ultra-high I/O.
+
+* `type` - (Optional, String) Specifies flavor type. The valid values are **single** and **cluster**.
+
+* `arch_type` - (Optional, String) Specifies the type of CPU architecture, e.g. **X86**.
+
+* `availability_zones` - (Optional, List) Specifies the list of availability zones with available resources.
+
+* `charging_mode` - (Optional, String) Specifies the flavor billing mode.
+  The valid values are **prePaid** and **postPaid**.
```

Comment:
The billing modes are all `postPaid` and this field is not required.

Reply:
done, fix it